packages/frontend/src/common/components/Footer.tsx

```diff
@@ -25,15 +25,15 @@ export default function Footer() {
           marginTop={theme.spacing(1)}
           fontSize="0.8rem"
         >
-          <Link href="/privacy" underline="hover">
+          <Link href="/privacy" underline="hover" target="_blank" rel="noopener">
```

Comment:
I changed all links to open as _blank to prevent the user from accidentally closing boards